.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+* [ ] I have read the [guidelines].
+
+<!--
+    We categorize our issues into 2 categories:
+    * Bug: Prefix your issue title with [BUG]
+    * Feature Request: Prefix your issue title with [FEATURE]
+    * Question: Prefix your issue title with [QUESTION]
+-->
+
+<!--
+    This section is only for Bug and Question issues
+-->
+* Sponge Implementation: <!-- SpongeForge or SpongeVanilla -->
+* Sponge build version: <!-- Find the version using /sponge version -->
```

Comment:
> Actual @dualspiral most server owners should know which Sponge flavor they are using and the build

Maybe now whilst Sponge is still rapidly evolving, but once it stabilises more and recommended builds become more important, I can see that some server owners won't know this information - they'll just be clicking the big shiny "Download" button. It happened with Bukkit too.

I see the point about `/sponge version`. In that case, just ask them to copy and paste that information. Some server owners may not quite know what you mean by "implementation" and "build version".

It may be better to at least add "if you aren't sure, _copy and paste_ the output of `/sponge version` here". Make it explicit.